Dockerfile

```diff
@@ -36,7 +36,7 @@ WORKDIR /install
 # When updating, update the README and both alpine_version ARGs
 #  * Use current version from https://alpinelinux.org/downloads/
 #  * ARGs repeat because Dockerfile ARGs are layer specific
-ARG alpine_version=3.14.2
```

Comment:
I think if you declare this arg on top (before from) and then declare on every from without value, the value can be reused.

Reply:
Thanks. I will give it a try.